deploy/kubernetes/helm/che/templates/dashboard-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: che
+    component: che-dashboard
+  name: che-dashboard
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: che
+      component: che-dashboard
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: che
+        component: che-dashboard
+    spec:
+      containers:
+      - name: che-dashboard
+        image: {{ .Values.cheDashboardImage }}
+        imagePullPolicy: {{ .Values.cheImagePullPolicy }}
+        ports:
+          - containerPort: 8080
+            name: http
+        resources:
+          limits:
+            memory: 256Mi
+          requests:
```

Comment:
what about 
```
    resources:                                                                                                                                                                                               │
│       limits:                                                                                                                                                                                                │
│         cpu: 500m                                                                                                                                                                                            │
│         memory: 256Mi                                                                                                                                                                                        │
│       requests:                                                                                                                                                                                              │
│         cpu: 50m                                                                                                                                                                                             │
│         memory: 16Mi
```

Reply:
:+1: I think it makes sense to apply the same changes as plugin/devfile registries have

Reply:
The recommended settings are applied in https://github.com/eclipse/che/pull/18093/commits/dd7bc79787ebfb906bde4b0be4513ea269b63f5f